c_emulator/riscv_platform.c

```diff
@@ -108,7 +108,7 @@ mach_bits plat_rom_size(unit u)
 }
 
 // Provides entropy for the scalar cryptography extension.
-mach_bits plat_get_16_random_bits()
+mach_bits plat_get_16_random_bits(unit u)
```

Comment:
Will this give warnings about unused variables? I think you can just omit the ` u`.

Reply:
I think you would need `__attribute__((unused)) unit u`, but I don't think we have that warning enabled for that file.

Reply:
You can just remove the `u` so it's `mach_bits plat_get_16_random_bits(unit)`, or add `(void)u;` in the body, but fair point about not having those warnings enabled.

Reply:
That would create another warning, except this time in the other direction because omitting the parameter name is a C2x extension! 😄 
```
c_emulator/riscv_platform.c:111:39: warning: omitting the parameter name in a function definition is a C2x extension [-Wc2x-extensions]
mach_bits plat_get_16_random_bits(unit)
```

Reply:
Ha TIL - I pretty much always use C++.